os/board/rtl8721csm/src/component/os/tizenrt/rtk_netmgr.c

```diff
@@ -318,6 +319,18 @@ trwifi_result_e wifi_netmgr_utils_scan_ap(struct netdev *dev, trwifi_ap_config_s
 	return wuret;
 }
 
+trwifi_result_e wifi_netmgr_utils_single_channel_scan(struct netdev *dev, trwifi_scan_config_s *scan_config)
+{
+	trwifi_result_e wuret = TRWIFI_FAIL;
+	if(cmd_wifi_scan_with_ssid(scan_config) != RTW_SUCCESS){

```

Comment:
Could you add whitespaces after "if" and ")" please

Reply:
Done